src/lib/awsManager.js

```diff
@@ -216,9 +302,11 @@ module.exports = {
   constructParamPath,
   getParameter,
   getParametersByService,
+  getSharedConfigByService,
+  putSharedConfigRecordsByService,
   setParameter,
   setParametersByService,
   getEnvironments,
   getServicesForEnvironment,
   getAllOrgParams
-}
```

Comment:
If time permits, you could add tests too for the 2 new commands - there already exists tests for the current ones

Reply:
(only asking because the tests for this module, if I remember correctly, are complete - and it would be best to maintain it going forward)

Reply:
tests added 👍